blocks/quote/quote.js

```diff
@@ -0,0 +1,38 @@
+import { createOptimizedPicture, decorateIcons } from '../../scripts/aem.js';
+import { span } from '../../scripts/dom-helpers.js';
+
+export default function decorate(block) {
+  /* change to ul, li */
+  const ul = document.createElement('ul');
```

Comment:
if you use the dom-helpers you can add the li elements directly in the constructor, with something like:
```
ul(...[block.children].map((row) => <row manipulation logic>)));
```